### PR TITLE
feat: introduces github templates

### PR DESCRIPTION
**Pull request by bartoszmajsak**
Since this year GitHub allows to have multiple templates [1] for creating issues. In May they enabled mapping between template files and issue types, so when user is creating one they can select one of the predefined types [2]. Once we merge this PR we would have to set the mappings.

This PR provides:

  * new bug template based on what we have in Arquillian community
  * enables incident report to be one of the issue templates too (@joshuawilson we might want to rename it as bug and incident might be confusing for external users)

There's also `uxd.md` template which was never used I believe but we can enable it and let UXD team customize based on their needs (cc @catrobson).

[1] https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/
[2] https://blog.github.com/2018-05-02-issue-template-improvements/

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<!--
+Thanks for interest in Openshift.io and apologies for making your experience not yet as awesome as we strive for.  
+
+Please follow template provided below to report a bug you have encountered and will make sure to improve it.
```
Comment (joshuawilson) [inline .github/ISSUE_TEMPLATE/bug.md]: `and will make sure to improve it` has a typo. It needs a `we` in it. `and we will make sure to improve it`

File: .github/ISSUE_TEMPLATE/incident.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+An Incident report should be created for each OSiO outage(small or large) in GitHub with label "type/incident":
+https://github.com/openshiftio/openshift.io/issues
+
+Note: this is a public report so de sensitise data. No tokens, no users, no internal links etc
+
+-->
+
+## Incident title
+
+*Period*: "From-To date"
```
[the remark this answers is not in the document]
Comment (bartoszmajsak) [reply .github/ISSUE_TEMPLATE/incident.md]: Nothing changed here - it's your document, just changed to `.md`

File: .github/ISSUE_TEMPLATE/incident.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<!-- 
+
+The incident report is intended as a write up for transparency, cross-team knowledge transfer and to
+get in the habit of thinking in terms of continues improvement.
+
+An Incident report should be created for each OSiO outage(small or large) in GitHub with label "type/incident":
+https://github.com/openshiftio/openshift.io/issues
+
+Note: this is a public report so de sensitise data. No tokens, no users, no internal links etc
```
[the remark this answers is not in the document]
Comment (bartoszmajsak) [reply .github/ISSUE_TEMPLATE/incident.md]: Nothing changed here - it's your document, just changed to .md

And really - it's open for reviewers to edit - you are more than welcome to fix it. Takes faster than typing this.

[the remark this answers is not in the document]
Comment (bartoszmajsak) [reply .github/ISSUE_TEMPLATE/incident.md]: I think you meant here `de-sensitise`

File: .github/ISSUE_TEMPLATE/incident.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshuawilson) [reply .github/ISSUE_TEMPLATE/incident.md]: I misread it. It is correct.

**Pull request merged**